src/ontology/mondo-ingest.Makefile

```diff
@@ -598,24 +599,26 @@ $(REPORTDIR)/%.subclass.confirmed.robot.tsv $(REPORTDIR)/%.subclass.added.robot.
 	--mondo-mappings-path $(TMPDIR)/mondo.sssom.tsv \
 	--onto-config-path metadata/$*.yml
 
+# Synchronization: Synonyms
 # todo: -deleted case to be added eventually
-# todo: when done with development, remove 'tmp/synonym_sync_combined_cases.tsv' and replace goal below
 .PHONY: sync-synonyms
-sync-synonyms: tmp/synonym_sync_combined_cases.tsv
-#sync-synonyms: $(REPORTDIR)/sync-synonyms.added.tsv $(REPORTDIR)/sync-synonyms.confirmed.tsv $(REPORTDIR)/sync-synonyms.updated.tsv
+sync-synonyms: tmp/synonym_sync_combined_cases.tsv $(REPORTDIR)/sync-synonyms.added.tsv $(REPORTDIR)/sync-synonyms.confirmed.tsv $(REPORTDIR)/sync-synonyms.updated.tsv
 
 tmp/mondo-synonyms-scope-type-xref.tsv:
 	$(MAKE) up-to-date-mondo.owl
-	$(ROBOT) query -i tmp/mondo-edit.owl --query ../sparql/mondo-synonyms-scope-type-xref.sparql $@
+	$(ROBOT) query -i tmp/mondo.owl --query ../sparql/synonyms-scope-type-xref.sparql $@
 
-../../tests/input/sync_synonym/mondo-synonyms-scope-type-xref.tsv: ../../tests/input/sync_synonym/test_mondo.owl
-	$(ROBOT) query -i $< --query ../sparql/mondo-synonyms-scope-type-xref.sparql $@
+tmp/%-synonyms-scope-type-xref.tsv: $(COMPONENTSDIR)/%.owl
+	$(ROBOT) query -i $(COMPONENTSDIR)/$*.owl --query ../sparql/synonyms-scope-type-xref.sparql $@
+
+../../tests/input/sync_synonym/%-synonyms-scope-type-xref.tsv:
+	$(ROBOT) query -i ../../tests/input/sync_synonym/test_$*.owl --query ../sparql/synonyms-scope-type-xref.sparql $@
 
 tmp/mondo-excluded-synonyms.tsv:
 	$(MAKE) up-to-date-mondo-edit.owl
 	$(ROBOT) query -i tmp/mondo-edit.owl --query ../sparql/mondo-excluded-synonyms.sparql $@
 
-# todo: temp output for analysis during development
+# todo: temp output for analysis during development; at the end, remove it and its usages
```

Comment:
This file is what gets uploaded as the [google sheet](https://docs.google.com/spreadsheets/d/1ftEZ47-JDe-1Z_fRWyO5NS39qT_-8MZsHXHEKz9aZ9Q/edit?gid=467804832#gid=467804832).

Trish asked to move this to `reports/` instead of `tmp/`.

It's unclear now, but it may be that this is something we want to keep permanently.